_data/civic_hackers.yml

```diff
@@ -109,6 +121,7 @@ Civic Hackers:
   - opennorth-archive
   - openopps
   - openplans
+  - opensaltlake
```

Comment:
Failing CI. Looks like it's now codeforutah https://www.codeforutah.org/about

> Open Salt Lake no longer encompassed the core population of technologists in Utah nor the range of community work it should be embracing.
>
> Rebranding as Code for Utah

Reply:
Yeah, I'm trying to figure out why the script says it's OK, but the org doesn't exist

Reply:
OK, I manually removed it, I guess that the script is pushing orgs out without checking if they're still valid